styles/styles.css

```diff
@@ -25,6 +25,9 @@
   --heading-font-family: var(--body-font-family);
   --fixed-font-family: "Spezia_SemiMono", menlo, consolas, 'Liberation Mono', monospace;
 
+   /* font icon aliases */
+   --vlt-right-arrow: "\e90c";
```

Comment:
good idea! 👏

Reply:
Yup!  If I get a moment I can find all the instances of icons in the font pack, and generate suitable human readable names for all of them to make them easier to leverage going forward (maybe will spin off a quick branch to do that)